src/client/features/enrichment/token-input.js

```diff
@@ -24,6 +24,12 @@ class TokenInput extends React.Component {
     let { inputBoxContents } = this.state;
     let { controller } = this.props;
 
+    if(!inputBoxContents) return;
```

Comment:
Why does inputBoxContents have to initially come from the parent props?

Reply:
That's an old part from before the refactoring. I just changed it now since its no longer needed.

Reply:
Ok so you can remove this if statement now